README.md

```diff
@@ -4,7 +4,34 @@ In this repository, we collect example scripts that can be useful for the team a
 
 ## Prerequisites
 
-TBD
+To run python scripts inside `examples` directory you should first run the installer script:
+
+```bash
+$ bin/install
+```
+
+This will handle installing `venv` and requierd packages.
```

Comment:
I would mention the `.config` file here.

Reply:
I mentioned it in the parent README since handling authorization is a generic thing but I added a refrence to it here

Reply:
Sorry, could you please explain the "parent README" ? This README file is in the root of the repository. I can't see the reference at line #13

Reply:
@kasra-r77 any chance to fix that?

Reply:
done